ClientApp/src/components/reuseables/ProjectEvaluationResults/EvaluationNotesTable.tsx

```diff
@@ -23,20 +22,20 @@ export const EvaluationNotes: React.FC<EvaluationNotesProps> = ({
         </Row>
       </Card.Header>
       <ListGroup variant="flush">
-        {notes.map(
+        {notes?.map(
           ({
-            noteId: id,
-            note,
-            employeeReviewer: { firstName, lastName },
-          }: ProjectEvaluationEmployeeNote): JSX.Element => (
-            <ListGroup.Item key={id}>
+            content,
+            notesId,
+            employee: { fullName } = {},
+          }: EvaluationNote): JSX.Element => (
+            <ListGroup.Item key={notesId}>
               <Row className="d-flex align-items-center">
                 <Col>
-                  <h6>{fullName(firstName, lastName)}</h6>
+                  <h6>{fullName}</h6>
                 </Col>
               </Row>
               <Row className="d-flex align-items-center">
-                <Col>{note || "-"}</Col>
+                <Col>{content || "-"}</Col>
```

Comment:
If back end team isn't able to update what's returned for no notes, then this should be updated to:
```suggestion
                <Col>{content !== "NA" ? content || "-" : "-"}</Col>
```
But it would be better for the back end to change since doing it this way would ignore an edge case where a user actually types "NA" into the notes field.

Reply:
Backend will change it.